src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -1029,6 +1030,10 @@ bool fuse_type_to_constant(const std::shared_ptr<ngraph::Node>& node,
                            const precisions_map& precisions,
                            const std::vector<Input<Node>>& consumers) {
     auto from = node->get_element_type();
+    // Consts marked with disable_constant_folding should be kept in f16 until they reach to the plugin
+    if (constant_folding_is_disabled(node))
+        return false;
```

Comment:
Are we sure this only affects CPU and other plugins get the same graph as before?

Reply:
for constants now i use a separate `rt_info` to ensure we don't interfere with other transformations markings